bot/src/alert.js

```diff
@@ -1,37 +1,19 @@
 const { listSubscriptions } = require('./db');
 const { TELEGRAM_TOKEN } = require('./environment');
+const axios = require('axios');
 const https = require('https');
 
-function alert(name, message) {
-    listSubscriptions(name).then((subss) => {
-        let sendMessagePath = `bot${TELEGRAM_TOKEN}/sendMessage`;
-
-        subss.forEach((subs) => {
-            let req = https.request({
-                host: "https://api.telegram.org/",
-                path: sendMessagePath,
-                method: "POST",
-                headers: {
-                    "Content-Type": "application/json"
-                }
-            }, (res) => {
-                if (res.statusCode == 200) {
-                    console.log(`Sent message to '${subs}'`);
-                } else {
-                    console.log(`Couldn't send message to '${subs}'`);
-                }
-            });
-
-            req.write(JSON.stringify({
-                chatId: subs,
-                text: message
-            }));
-
-            req.end();
-
-        });
+async function alert(name, message) {
```

Comment:
@gutorc92, have you tested the async/await functions with the DialogFlow lib we are using? If not, please do so and let me know if everything ran smoothly.

Reply:
I had make some tests, it works.